Index: CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/controllers/EventsController.java
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/controllers/EventsController.java b/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/controllers/EventsController.java
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/controllers/EventsController.java	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,2 +0,0 @@
-package org.timetable.controllers;public class EventsController {
-}
Index: CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/MySpringMvcDispatcherServletInitializer.java
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/MySpringMvcDispatcherServletInitializer.java b/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/MySpringMvcDispatcherServletInitializer.java
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/MySpringMvcDispatcherServletInitializer.java	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,22 +0,0 @@
-package org.timetable.config;
-
-import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
-
-public class MySpringMvcDispatcherServletInitializer extends
-        AbstractAnnotationConfigDispatcherServletInitializer {
-
-    @Override
-    protected Class<?>[] getRootConfigClasses() {
-        return null;
-    }
-
-    @Override
-    protected Class<?>[] getServletConfigClasses() {
-        return new Class[] {SpringConfig.class};
-    }
-
-    @Override
-    protected String[] getServletMappings() {
-        return new String[] {"/"};
-    }
-}
Index: CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/WEB-INF/web.xml
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/WEB-INF/web.xml b/CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/WEB-INF/web.xml
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/WEB-INF/web.xml	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,7 +0,0 @@
-<!DOCTYPE web-app PUBLIC
- "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN"
- "http://java.sun.com/dtd/web-app_2_3.dtd" >
-
-<web-app>
-  <display-name>Archetype Created Web Application</display-name>
-</web-app>
Index: CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/domain/Event.java
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/domain/Event.java b/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/domain/Event.java
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/domain/Event.java	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,2 +0,0 @@
-package org.timetable.domain;public class Event {
-}
Index: CrosslinkTimetable/src/backend/TimetableAPI/pom.xml
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/pom.xml b/CrosslinkTimetable/src/backend/TimetableAPI/pom.xml
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/pom.xml	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,103 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-
-  <groupId>org.example</groupId>
-  <artifactId>TimetableAPI</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <packaging>war</packaging>
-
-  <name>TimetableAPI Maven Webapp</name>
-  <!-- FIXME change it to the project's website -->
-  <url>http://www.example.com</url>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
-    <spring.version>5.2.1.RELEASE</spring.version>
-  </properties>
-
-  <dependencies>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.11</version>
-      <scope>test</scope>
-    </dependency>
-
-    <!-- https://mvnrepository.com/artifact/org.springframework/spring-core -->
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-core</artifactId>
-      <version>${spring.version}</version>
-    </dependency>
-
-    <!-- https://mvnrepository.com/artifact/org.springframework/spring-context -->
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-context</artifactId>
-      <version>${spring.version}</version>
-    </dependency>
-
-    <!-- https://mvnrepository.com/artifact/org.springframework/spring-web -->
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-web</artifactId>
-      <version>${spring.version}</version>
-    </dependency>
-
-    <!-- https://mvnrepository.com/artifact/org.springframework/spring-webmvc -->
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-webmvc</artifactId>
-      <version>${spring.version}</version>
-    </dependency>
-
-    <!-- https://mvnrepository.com/artifact/org.thymeleaf/thymeleaf-spring5 -->
-    <dependency>
-      <groupId>org.thymeleaf</groupId>
-      <artifactId>thymeleaf-spring5</artifactId>
-      <version>3.0.11.RELEASE</version>
-    </dependency>
-  </dependencies>
-
-  <build>
-    <finalName>TimetableAPI</finalName>
-    <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
-      <plugins>
-        <plugin>
-          <artifactId>maven-clean-plugin</artifactId>
-          <version>3.1.0</version>
-        </plugin>
-        <!-- see http://maven.apache.org/ref/current/maven-core/default-bindings.html#Plugin_bindings_for_war_packaging -->
-        <plugin>
-          <artifactId>maven-resources-plugin</artifactId>
-          <version>3.0.2</version>
-        </plugin>
-        <plugin>
-          <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.8.0</version>
-        </plugin>
-        <plugin>
-          <artifactId>maven-surefire-plugin</artifactId>
-          <version>2.22.1</version>
-        </plugin>
-        <plugin>
-          <artifactId>maven-war-plugin</artifactId>
-          <version>3.2.2</version>
-        </plugin>
-        <plugin>
-          <artifactId>maven-install-plugin</artifactId>
-          <version>2.5.2</version>
-        </plugin>
-        <plugin>
-          <artifactId>maven-deploy-plugin</artifactId>
-          <version>2.8.2</version>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-  </build>
-</project>
Index: CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/outlook_exchange/EventExchange.java
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/outlook_exchange/EventExchange.java b/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/outlook_exchange/EventExchange.java
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/outlook_exchange/EventExchange.java	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,5 +0,0 @@
-package org.timetable.outlook_exchange;
-
-public class EventManager {
-
-}
Index: CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/SpringConfig.java
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/SpringConfig.java b/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/SpringConfig.java
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/java/org/timetable/config/SpringConfig.java	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,50 +0,0 @@
-package org.timetable.config;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-import org.springframework.web.servlet.config.annotation.ViewResolverRegistry;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
-import org.thymeleaf.spring5.SpringTemplateEngine;
-import org.thymeleaf.spring5.templateresolver.SpringResourceTemplateResolver;
-import org.thymeleaf.spring5.view.ThymeleafViewResolver;
-
-@Configuration
-@ComponentScan("org.timetable")
-@EnableWebMvc
-public class SpringConfig implements WebMvcConfigurer {
-
-    private final ApplicationContext applicationContext;
-
-    @Autowired
-    public SpringConfig(ApplicationContext applicationContext) {
-        this.applicationContext = applicationContext;
-    }
-
-    @Bean
-    public SpringResourceTemplateResolver templateResolver() {
-        SpringResourceTemplateResolver templateResolver = new SpringResourceTemplateResolver();
-        templateResolver.setApplicationContext(applicationContext);
-        templateResolver.setPrefix("/WEB-INF/views/");
-        templateResolver.setSuffix("");
-        return templateResolver;
-    }
-
-    @Bean
-    public SpringTemplateEngine templateEngine() {
-        SpringTemplateEngine templateEngine = new SpringTemplateEngine();
-        templateEngine.setTemplateResolver(templateResolver());
-        templateEngine.setEnableSpringELCompiler(true);
-        return templateEngine;
-    }
-
-    @Override
-    public void configureViewResolvers(ViewResolverRegistry registry) {
-        ThymeleafViewResolver resolver = new ThymeleafViewResolver();
-        resolver.setTemplateEngine(templateEngine());
-        registry.viewResolver(resolver);
-    }
-}
\ No newline at end of file
Index: CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/index.jsp
===================================================================
diff --git a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/index.jsp b/CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/index.jsp
deleted file mode 100644
--- a/CrosslinkTimetable/src/backend/TimetableAPI/src/main/webapp/index.jsp	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
+++ /dev/null	(revision 473ebb0299eb1ad1580a839fd3250de655e4ad43)
@@ -1,5 +0,0 @@
-<html>
-<body>
-<h2>Hello World!</h2>
-</body>
-</html>
